client/tests/integration/components/project-geography-test.js

```diff
@@ -0,0 +1,100 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, click, fillIn } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import EmberObject from '@ember/object';
+
+module('Integration | Component | project-geography', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('user can add a new bbl', async function(assert) {
+    const ourModel = EmberObject.extend({});
+
+    // array of objects
+    // bbls array is dcp_dcp_projectbbl_dcp_pasform
+    const pasFormModel = ourModel.create({
```

Comment:
Can you try to make this test work without using EmberObject? I think POJOs would work fine here.